docker/ci/Dockerfile

```diff
@@ -14,16 +14,15 @@ WORKDIR "/tmp"
 RUN wget -qO ${BOOST_BASENAME}.tar.gz ${BOOST_URL} && \
     tar xzf ${BOOST_BASENAME}.tar.gz && \
     cd ${BOOST_BASENAME} && \
-    ./bootstrap.sh && \
+    ./bootstrap.sh --with-libraries=thread,log,filesystem,program_options && \
```

Comment:
A better change would probably be to use the "`bootstrap_boost.sh`" script in the `util/build_prep` directory which has a `-m` option to take care of this

Reply:
Thanks for the tip about using the -m option, exactly what I needed.